packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -1078,7 +1078,7 @@ def test_if_minuteman_routes_to_vip(cluster, timeout=125):
         'healthChecks': [{
             'protocol': 'COMMAND',
             'command': {
-                'value': 'test "$(curl -o /dev/null --max-time 5 -4 -w \'%{http_code}\' -s http://localhost:${PORT0}/|cut -f1 -d" ")" == 200'  # noqa
+                'value': 'curl -f -X GET http://$HOST:$PORT0/'
```

Comment:
I like the simplified health check, but why is this changing? Would be nice to have some more details in the commit message.